managed-ledger/pom.xml

```diff
@@ -84,7 +84,16 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+       <groupId>io.dropwizard.metrics</groupId>
```

Comment:
Can you explain why do we need this? I failed to see this dependency was used here. The same question apply to the other dependency.

Reply:
because from zk 3.6 onwards the "zookeeper" Maven artifact does not import those dependencies anymore to the dependant projects.
This is because we (ZooKeeper project community) want to say that this artifact is only the "zookeeper java client" and the zookeeper java client does not need such jars.
But we still do not provide a "zookeeper server" jar

In Pulsar we are also running the ZooKeeper server (both for production and for tests) and so we need to explicitly add those two jar to the build, otherwise the ZooKeeper server won't start

Reply:
It would be good to leave a comment with the reason we're adding these, otherwise down the road it will be difficult to remember the rationale.

Reply:
@sijie  PTAL the description was added on the new deps